src/averageAges.js

```diff
@@ -37,7 +47,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const motherList
+  = people.filter(item => item.mother).map(item => item.mother);
```

Comment:
I don't really think it's a good idea to name people `item`🙂
Try `person` instead